src/Pos/Launcher/Runner.hs

```diff
@@ -344,14 +345,14 @@ setupLoggers params = setupLogging =<< getRealLoggerConfig params
 loggerBracket :: LoggingParams -> IO a -> IO a
 loggerBracket lp = bracket_ (setupLoggers lp) releaseAllHandlers
 
+-- TODO: remove this function and all it's calls?
 addDevListeners
     :: MinWorkMode m => Timestamp
     -> ListenersWithOut m
     -> ListenersWithOut m
 addDevListeners sysStart ls =
     if Const.isDevelopment
-    then mergeLs [ stubListenerOneMsg (Proxy :: Proxy SysStartResponse)
-                 , sysStartReqListener sysStart] `mappendPair` ls
+    then mergeLs [ stubListenerOneMsg (Proxy :: Proxy MsgGetHeaders)] `mappendPair` ls
```

Comment:
1. There is
>  +-- TODO: remove this function and all it's calls? 
Do you know the answer?
2. How did `MsgGetHeaders` appear here? It's very counterintuitive.